ReactApp/.storybook/Wrapper.jsx

```diff
@@ -0,0 +1,117 @@
+import React, { Component } from "react";
+import { BrowserRouter } from "react-router-dom";
+import "typeface-roboto";
+import {
+  ThemeProvider,
+  StyledEngineProvider,
+  createTheme,
+} from "@mui/material/styles";
+import CssBaseline from "@mui/material/CssBaseline";
+
+import AutomationStudioContext from "React-Automation-Studio/components/SystemComponents/AutomationStudioContext";
+
+import { io } from "socket.io-client";
+import lightTheme from "React-Automation-Studio/components/UI/Themes/lightTheme";
+import ReactVisCssBaseline from "React-Automation-Studio/components/SystemComponents/ReactVisCssBaseline";
+
+if (typeof window.socket === "undefined") {
+  window.socket = io("/pvServer", {
+    transports: ["websocket"],
+  });
+}
+
+const themes = { Light: lightTheme };
+class Wrapper extends Component {
```

Comment:
Is use of classes mandatory for storybook? It will work. That is clear. Yet, it seems to me the solution converges towards hooks. Or, is this just false impression?